nx-X11/programs/Xserver/hw/nxagent/Display.c

```diff
@@ -1838,43 +1834,56 @@ static FILE *nxagentLookForIconFile(char *iconName, const char *permission,
     return NULL;
   }
 
-  for(breakLoop = 0; breakLoop == 0 && fptr == NULL; )
+  for (int breakLoop = False; breakLoop == False && fptr == NULL; )
```

Comment:
Can drop `breakLoop` by using:

```C
  for (char *end = path; end == NULL && fptr == NULL; )
```

Reply:
No, I'd have to check for `end != NULL && fprt == NULL`. Done.

Reply:
Yes, sorry.